echopype/tests/echodata/test_echodata.py

```diff
@@ -168,3 +168,12 @@ def test_update_platform():
 
     for variable in updated:
         assert not np.isnan(ed.platform[variable].values).all()
+
+    # check times are > min(ed.beam["ping_time"]) - 2s
+    assert (ed.platform["location_time"] > ed.beam["ping_time"].min() - np.timedelta64(2, "s")).all()
+    # check there is only 1 time < min(ed.beam["ping_time"])
+    assert np.count_nonzero(ed.platform["location_time"] < ed.beam["ping_time"].min()) == 1
+    # check times are < max(ed.beam["ping_time"]) + 2s
+    assert (ed.platform["location_time"] < ed.beam["ping_time"].max() + np.timedelta64(2, "s")).all()
+    # check there is only 1 time > max(ed.beam["ping_time"])
+    assert np.count_nonzero(ed.platform["location_time"] > ed.beam["ping_time"].max()) == 1
```

Comment:
Where does the 2s come from? Is that the temporal interval of the data in `extra_platform_data`? It would be good to add the explanation to the comments so people know why there's a hard-code value. (I have not tried to look into what's in the extra_platform_data and ed.beam.ping_time.)

Reply:
Yes that is the interval of the data